hdmi/tinyaudio_hw.c

```diff
@@ -228,25 +232,23 @@ static int start_output_stream(struct stream_out *out)
     struct audio_device *adev = out->dev;
     struct pcm_params *params;
     int device = 0;
-
+    
     ALOGV("%s enter",__func__);
 
     if ((adev->card < 0) || (adev->device < 0)){
         char value[PROPERTY_VALUE_MAX];
-
+ 
         /*this will be updated once the hot plug intent
           sends these information.*/
-        adev->card = DEFAULT_CARD;
+        adev->card = DEFAULT_CARD; 
+        adev->device = parse_hdmi_device_number();
+        /*Keeping the condtion check only for EHL, as it is not verified*/   
         property_get("ro.vendor.hdmi.audio", value, "0");
         if (!strcmp(value,"ehl")) {
             adev->device = DEFAULT_DEVICE_EHL;
-        } else {
-            adev->device = DEFAULT_DEVICE;
+            ALOGV("%s : Setting default card/ device %d,%d",__func__,adev->card,adev->device);
```

Comment:
introduce a space between , and __func__ . Same comment applies for , and adev->card etc